project/build.scala

```diff
@@ -125,12 +136,12 @@ object BananaRdfBuild extends Build {
   lazy val rdfTestSuite_jvm = rdfTestSuiteM
     .project(Jvm, rdfTestSuite_common_jvm)
     .settings(Seq(resourceDirectory in Test := baseDirectory.value / "src/main/resources"):_*)
-    .dependsOn(rdf_jvm)
+    .dependsOn(rdf_jvm, ntriples_jvm)
```

Comment:
> why does the test suite depend on ntriples? 

Because it always has :) - [see](
https://github.com/w3c/banana-rdf/commit/55d186a771ac0c63c971d63a589b6820cd2665e8#diff-798d1bfac5c6765ba2f0c34e37c073ed)

But for sure it can be removed here. Just looking at the others, some of which will _definitely_ be needed